domain/src/main/java/no/difi/meldingsutveksling/receipt/MessageStatus.java

```diff
@@ -1,61 +1,98 @@
 package no.difi.meldingsutveksling.receipt;
 
 import com.fasterxml.jackson.annotation.JsonFormat;
-import com.google.common.base.MoreObjects;
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonView;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Getter;
+import lombok.Setter;
+import no.difi.meldingsutveksling.nextmove.AbstractEntity;
+import no.difi.meldingsutveksling.view.Views;
 
 import javax.persistence.*;
-import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
 
 /**
  * Used for storing and tracking receipt information.
  */
+@Getter
+@Setter
 @Entity
-@Data
-public class MessageStatus {
-
-    @Id
-    @GeneratedValue
-    private Integer statId;
-    @Column(insertable = false, updatable = false, name = "conv_id")
-    private Integer convId;
-    private String conversationId;
+@Table(name = "message_status")
+@NamedEntityGraph(name = "MessageStatus.conversation", attributeNodes = @NamedAttributeNode("conversation"))
+public class MessageStatus extends AbstractEntity<Long> {
+
+    @Override
+    @JsonProperty
+    @ApiModelProperty(
+            position = 2,
+            value = "Id",
+            example = "1")
+    @JsonView({Views.Conversation.class, Views.MessageStatus.class})
+    public Long getId() {
+        return super.getId();
+    }
+
+    @JsonProperty
+    @JsonView(Views.MessageStatus.class)
+    public Long getConvId() {
+        return conversation.getId();
+    }
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "conv_id")
+    @JsonIgnore
+    private Conversation conversation;
+
     @JsonFormat(shape = JsonFormat.Shape.STRING)
-    private LocalDateTime lastUpdate;
+    @JsonView({Views.Conversation.class, Views.MessageStatus.class})
+    private OffsetDateTime lastUpdate;
+    @JsonView({Views.Conversation.class, Views.MessageStatus.class})
     private String status;
+    @JsonView({Views.Conversation.class, Views.MessageStatus.class})
     private String description;
 
     @Lob
+    @JsonView(Views.MessageStatus.class)
     private String rawReceipt;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)